.drone.yml

```diff
@@ -95,21 +96,17 @@ pipeline:
       matrix:
         TEST_SUITE: acceptance
 
-# TODO codecov
-#  codecov:
-#    image: plugins/codecov:2
-#    secrets: [codecov_token]
-#    pull: true
-#    paths:
-#     - tests/clover.xml
-#    files:
-#     - '*.xml'
-#    when:
-#      matrix:
-#        TEST_SUITE: phpunit
-#        PHP_VERSION: 7.2
-#        DB_TYPE: mysql
-#        OC_VERSION: daily-master-qa
+  codecov:
+    image: plugins/codecov:2
+    secrets: [codecov_token]
+    pull: true
+    files:
+    - '*.xml'
```

Comment:
we need files and paths? looks wrong ...

Reply:
Oh, I copied it from `password_policy`, I will only use files then.